src/Components/Facility/LocationManagement.tsx

```diff
@@ -122,27 +122,36 @@ export const LocationManagement = (props: LocationManagementProps) => {
 
   if (locations && locations.length) {
     locationsList = locations.map((locationItem: LocationModel) => (
-      <LocationRow
-        id={locationItem.id || ""}
-        facilityId={facilityId || ""}
-        name={locationItem.name || ""}
-        description={locationItem.description || ""}
-      />
+      <div className="grow mt-5 bg-white p-4 flex flex-wrap">
+        <LocationRow
+          id={locationItem.id || ""}
+          facilityId={facilityId || ""}
+          name={locationItem.name || ""}
+          description={locationItem.description || ""}
+        />
+      </div>
     ));
   } else if (locations && locations.length === 0) {
     locationsList = (
-      <p className="bg-white flex justify-center text-2xl w-full font-bold px-5 py-5 border-b border-gray-200 text-center text-gray-500">
-        No locations available
-      </p>
+      <div className="h-full content-center grid justify-items-center mt-40">
+        <div className="flex justify-center w-full items-center">
+          <img
+            className="w-3/12 sm:w-3/12 md:w-3/12 lg:w-2/12 xl:w-2/12 2xl:w-2/12"
+            src={"https://cdn.coronasafe.network/location.png"}
```

Comment:
@siddnikh you use an icon instead of an image, also reduce the size of the icon a little

Reply:
sure, let me try making the image smaller.